### PR TITLE
[IMP] account_test : Migrate to new api

### PR DESCRIPTION
**Pull request by htr-odoo**
Task : https://www.odoo.com/web#id=12164&view_type=form&model=project.task&menu_id=3942&action=327
Pad : https://pad.odoo.com/p/r.h0opBFb0J2YKuyJ4

### CODE REVIEW

File: addons/account_test/account_test_data.xml [hunk 1]
```diff
@@ -151,5 +150,4 @@ if result:
   result.insert(0,_('* Here is the list of inactive partners and movement lines that are not correct: '))
```
Comment (mdi-odoo) [inline addons/account_test/account_test_data.xml]: Please change the test data according to model changes.
Currently it give's error while printing reports .

File: addons/account_test/report/account_test_report.py [hunk 2]
```diff
@@ -1,51 +1,21 @@
...
+# -*- coding: utf-8 -*-
 import datetime
-import time
-
-from openerp.osv import osv
-from openerp.tools.translate import _
-from openerp.report import report_sxw
-
-
+from openerp import _, models, api
```
Comment (mdi-odoo) [inline addons/account_test/report/account_test_report.py]: Please order your imports alphabetically !!!

File: addons/account_test/report/account_test_report.py [hunk 3]
```diff
@@ -86,9 +56,17 @@ def _format(item):
...
 
-
-class report_accounttest(osv.AbstractModel):
-    _name = 'report.account_test.report_accounttest'
-    _inherit = 'report.abstract_report'
-    _template = 'account_test.report_accounttest'
-    _wrapped_report_class = report_assert_account
+    @api.multi
+    def render_html(self, data=None):
+        report_obj = self.env['report']
```
Comment (mdi-odoo) [inline addons/account_test/report/account_test_report.py]: Use CamelCase for model object !!!

File: addons/account_test/report/account_test_report.py (hunk 2 above)
Comment (htr-odoo) [inline addons/account_test/report/account_test_report.py]: Done.

File: addons/account_test/report/account_test_report.py (hunk 3 above)
Comment (htr-odoo) [inline addons/account_test/report/account_test_report.py]: Done.

**Comment by mdi-odoo** [comment]
Now ready to merge.

**Comment by mdi-odoo** [comment]
Now ready to merge.

**Pull request merged**